operators/multiclusterobservability/pkg/config/azure_conf.go

```diff
@@ -4,9 +4,10 @@
 package config
 
 import (
-	"errors"
 	"strings"
 
```

Comment:
Why is there always an empty line between the other stdlib imports and the import of `"errors"`? This doesn't seem formatted with `goimports`.

Reply:
You're right, botched replace. Fixed it.